TestCases/serial_regression.py

```diff
@@ -1903,7 +1903,7 @@ def main():
     pywrapper_rigidMotion.cfg_dir       = "py_wrapper/flatPlate_rigidMotion"
     pywrapper_rigidMotion.cfg_file      = "flatPlate_rigidMotion_Conf.cfg"
     pywrapper_rigidMotion.test_iter     = 5
-    pywrapper_rigidMotion.test_vals     = [-1.551335, 2.295594, 0.350050, 0.093081]
+    pywrapper_rigidMotion.test_vals     = [-1.614170, 2.242953, 0.350050, 0.093137]
```

Comment:
these changed because the volumes at time n and time n-1 are now set equal to the initial value (changes in CDriver::GeometryPreprocessing and in CFVMFlowSolverBase<V, R>::PushSolutionBackInTime)

The other dynamic FSI cases changed because of that and because of initializing the grid-velocity-elasticity-problem.

Reply:
👍 for the explanation